docs/content/preview/yugabyte-voyager/release-notes.md

```diff
@@ -13,6 +13,30 @@ type: docs
 
 What follows are the release notes for the YugabyteDB Voyager v1 release series. Content will be added as new notable features and changes are available in the patch releases of the YugabyteDB v1 series.
 
+## v1.8.5 - November 11, 2024
+
+### Enhancements
+
+- The guardrails checks to validate source/target database permissions, verify binary dependencies, and check database version compatibility for PostgreSQL in all voyager commands is now enabled by default.
+- UI/UX improvements in the PostgreSQL permission grant script and new checks are added for replication slots, foreign keys, and triggers in PostgreSQL guardrails.
```

Comment:
UX improvements in a script?
What is this script? It is not referred to elsewhere in YBV docs, at least not using this name.

Reply:
Seems internal to me. @priyanshi-yb - can you clarify?